pom.xml

```
@@ -600,7 +600,7 @@ under the License.
         <!-- Use Scala 2.13 by default -->
         <scala.version>2.13.14</scala.version>
```

Scala was upgraded in the 4.0.0 release version:
```suggestion
        <scala.version>2.13.16</scala.version>
```